### DISCUSSION TITLE
Some Dags not running as scheduled

### DISCUSSION THREAD

**Discussion by erosons** [apache/airflow#30277] [Q&A]
Hello Community,

For three days consecutively, few  dags have refused to run as schedule, I practically have to manually trigger them everyday. 
I currently run a docker version of airflow and the server and scheduler are healthy. And idea what could be the possible cause.

**Comment by erosons** [comment]
I found the cause of the issue, I have two DAG scheduled at the exactly same  time, After varying the timing slight with minutes the DAGs ran. 

I am very curious to still know why DAGS set to run at the same exact will not run , Any idea will be approciated.

**Reply by potiuk** [reply]
Quite unlikely. You missed explanation on your version of Airlfow, and some details that could help to investigate it, but I doubt the cause is as simple (unless this is really old version that no-one remembers about)

**Reply by erosons** [reply]
My version is apache/airflow:2.2.4-python3.9

The situation was basically I set the runtime for two DAGs at exactly the same time. And for two consecutive days the Dags did not run, I have to go in and manually run it. After reading an experience of someone from stack overflow. I decide to vary the timing with few minutes.And the Dags started running again. Other DAGs were running fine at their schedule time, expect for the two Dags that has exact same time schedule. Hope that was detail enough

**Reply by potiuk** [reply]
Not really. We need logs from scheduler, showing the dags you had with details of their scheduling and more detailed information what happened there and everything else - detail of your deployment and easily reproducible case. Basically everything we ask you when you fill a new issue with Airflow: https://github.com/apache/airflow/issues/new/choose . There is a good reason why we ask for it.
Airlfow is a free OSS project run by volunteers and what you have been observing could com from a complety different reason than you think it comes from. It could be something else than same schedule for if you have.

Unless you can reproduce it and make sure that it is the diagnosis you have and give us a very clear and straightforward way to reproduce it (which you yourself verify as a reproducible case), no-one here voluntarily is not going to try to reproduce it based on the assumptions you had (especially that it is not very likely. And especially that it is based on such an old version of airflow - we had probably multiple hundreds bugs fixed since 2.2.4 (we are about to release 2.5.3). So I doubt somone would like to invest a lot of their free time for it. This is free software, you get with all the benefits (it costs nothing) but also it means that sometimes we need the users to be a bit more dilligent if they want (for example) understand why a version relased a year ago behaves like that in their specific circumstances. We are not releasing anything else than 2.5.* any more (and soon 2.6) so even if there is a bug, you will have to upgrade to latest one anyway if you would like to get it fixed (and you already found a workaround, that others could use in this case so even if it is a problem in 2.2.4. it's unlikely we are going to do anything about it.

If I may suggest it -> upgrade to 2.5.3 and try to reproduce it there. It is extremely likely you won't be able to reproduce it there. That will end the investigation. Problem fixed. But if you find it and are able to easily reproduce - please report it through issue with all the details and easily reproducible scenario.

This is the least our users can do in return for the free software they have.

**Reply by erosons** [reply]
Thank you
